stable/rbac-manager/ci/digest-values.yaml

```diff
@@ -0,0 +1,2 @@
+image:
+  digest: sha256testvalue
```

Comment:
Linter apparently doesn't like this lack of newline

Reply:
Oh, and I think we will need to put a valid sha256 for the end to end tests to pass here.